examples/lib/stories/effects/glow_effect_example.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:examples/commons/ember.dart';
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(GameWidget(game: GlowEffectExample()));
+}
+
+class GlowEffectExample extends FlameGame with TapDetector {
+  static const String description = '''
+    In this example we show how the `GlowEffect` can be used.
+  ''';
+
+  @override
+  Future<void> onLoad() async {
+    final paint2 = Paint()
+      ..color = const Color(0xff39FF14)
+      ..style = PaintingStyle.stroke;
+
+    add(
+      Ember(
```

Comment:
I think you can skip the ember here since the GlowEffect doesn't seem to work very well on it.